examples/cityscapes/README.md

```diff
@@ -0,0 +1,182 @@
+5 000 frames in addition to a larger set of 20 000 weakly annotated frames.
+
+### Latent Space Exploration
+The following plot illustrates a population exploration map, depicting the similarity among samples based on the latent 
+space of a trained model. This map is constructed using the extracted features of the model.
+A distinct separation is evident between images with high and low minimum values (min). This suggests that images with
+high min share common features that differ from those found in images with low min.
+
+![input_min](screen_shots/input_min.png)
+
+
+#### *Detecting & Handling High Loss Clusters*
+Upon conducting additional analysis, it was found that when the number of small bounding boxes decreases, 
+the object detection loss increases.
+
+![high_loss_small_bboxes](screen_shots/high_loss_small_bboxes.png)
+
+Furthermore, When the number of labeled persons decreases, the object detection loss increases. But not necessarily the 
+other direction is also true-There can be a variety of reasons why the loss is high.
+
+![high_loss_number_of_persons](screen_shots/high_loss_number_of_persons.png)
+
+By filtering the dot size in the latent space based on the image mean, we observe a cluster of samples that exhibit a 
+higher mean. The proximity of these samples suggests that they share similar features, indicating a potential 
+correlation between their characteristics.
+
+![high_loss_mean_image](screen_shots/high_loss_mean_image.png)
+
+#### *Mislabeled data*
+
+Upon conducting further analysis, it has been discovered that images containing cars exhibit a high loss. Subsequent 
+research has revealed that the algorithm is actually recognizing cars accurately, but the ground truth (gt) labels are 
+incorrect. The cars in the images are mislabeled. To address this issue, it is essential to correct the labels for the 
+cars in order to improve the performance of the algorithm.
+
+<div style="display: flex">
+  <img src="screen_shots/bb_gt_car.png" alt="Image 1" style="margin-right: 10px;">
+  <img src="screen_shots/bb_car.png" alt="Image 2" style="margin-left: 10px;">
+</div>
+
+We can observe that the class "motorcycle" is underrepresented in the data. The majority of images have zero labeled 
+ground truth bounding boxes for motorcycles. To address this, we should investigate if there are any mislabeling or 
+consider adding more images containing motorcycles to improve the representation of this class in the dataset.
+
+![Unlabeled_motorcycle](screen_shots/Unlabeled_motorcycle.png)
+
+
+#### *Detecting Unlabeled Clusters*
+
+Cluster number 14 of k-means clusters characterized in dark ambiance images, featuring a road surrounded by vegetation, 
+while cars are the primary objects present in the image.
+
+![cluster14_kmeans](screen_shots/cluster14_kmeans.png)
+
+### Sample Loss Analysis
+
+Within this section, we delve into the examination of features that impact the model's predictions. 
```

Comment:
Consider rephrasing this:

The label you chose to show is "LABEL_UP".
Isn't this basically showing what features have strong effects on the specific prediction (i.e. the one in the "building" logit)? 

If so suggest to rephrase the content. Otherwise suggest to change terminology in the second sentence to make it shorter (warmer, image -> prominent, figure).